Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -209,22 +247,42 @@ public class NavigationSettings {
                            navigatorPredictionInterval: TimeInterval? = nil,
                            liveIncidentsOptions: IncidentsOptions? = nil,
                            statusUpdatingSettings: StatusUpdatingSettings? = nil) {
+        self.initialize(
+            with: .init(
+                directions: directions,
+                tileStoreConfiguration: tileStoreConfiguration,
+                routingProviderSource: routingProviderSource,
+                alternativeRouteDetectionStrategy: alternativeRouteDetectionStrategy,
+                utilizeSensorData: utilizeSensorData,
+                navigatorPredictionInterval: navigatorPredictionInterval,
+                liveIncidentsOptions: liveIncidentsOptions,
+                statusUpdatingSettings: statusUpdatingSettings,
+                logLevel: .info
```

Comment:
Seems like different default values for `logLevel` in different methods: here it's `info`, in another init `debug` is used. 
It seems like it would be better to set the default value in one place and use `info` as it was used before changes